.github/workflows/gateway-conformance.yml

```diff
@@ -0,0 +1,86 @@
+name: Gateway Conformance
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.event_name == 'push' && github.sha || github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  gateway-conformance:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      # 1. Download the gateway-conformance fixtures
+      - name: Download gateway-conformance fixtures
+        uses: ipfs/gateway-conformance/.github/actions/extract-fixtures@v0.0
+        with:
+          output: fixtures
+
+      # 2. Build the kubo-gateway
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.x
+      - name: Checkout kubo-gateway
+        uses: actions/checkout@v3
+        with:
+          path: kubo-gateway
+      - name: Build kubo-gateway
+        run: make build
+        working-directory: kubo-gateway
+
+      # 3. Start the kubo-gateway
+      - name: Start kubo-gateway
+        env:
+          GATEWAY_PUBLIC_GATEWAYS: |
+            {
+              "example.com": {
+                "UseSubdomains": true,
+                "Paths": ["/ipfs", "/ipns", "/api"]
+              },
+              "localhost": {
+                "UseSubdomains": true,
+                "InlineDNSLink": true,
+                "Paths": ["/ipfs", "/ipns", "/api"]
```

Comment:
Maybe we can remove the `/api`? That's not part of the Gateway specification anyways and they are not being tested.